Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Management/API/Driver.yaml

```diff
@@ -0,0 +1,30 @@
+imports:
+  ServiceNames: Dashboard.ProviderPlatform.Driver
+  Currency: Kernel.Types.Common
+  HighPrecMoney: Kernel.Types.Common
+
+module: Driver
+
+apis:
+- POST:
+    endpoint: /clearFee
+    auth: ApiAuth DRIVER_OFFER_BPP_MANAGEMENT DRIVERS CLEAR_FEE
+    mandatoryQuery:
+      driverId: Text
```

Comment:
it can be Id Driver:
```
Driver: Dashboard.Common
............
mandatoryQuery:
  driverId: Id Driver
```

Reply:
when I use
   ```
 mandatoryQuery:
      driverId: Id Driver  
```
the dsl is generating following api type :-
```
type PostDriverClearFee =
  ( "clearFee" :> MandatoryQueryParam "driverId" Kernel.Types.Id.Id Dashboard.ProviderPlatform.Driver.Driver
      :> ReqBody
           '[JSON]
           API.Types.ProviderPlatform.Management.Driver.ClearDriverFeeReq
      :> Post '[JSON] Kernel.Types.APISuccess.APISuccess
  )
```
this causing compilation issue , should be like :-
```
type PostDriverClearFee =
  ( "clearFee" :> MandatoryQueryParam "driverId" (Kernel.Types.Id.Id Dashboard.ProviderPlatform.Driver.Driver)
      :> ReqBody
           '[JSON]
           API.Types.ProviderPlatform.Management.Driver.ClearDriverFeeReq
      :> Post '[JSON] Kernel.Types.APISuccess.APISuccess
  )
```
or
```
type PostDriverClearFee =
  ( "clearFee" :> MandatoryQueryParam "driverId" $ Kernel.Types.Id.Id Dashboard.ProviderPlatform.Driver.Driver
      :> ReqBody
           '[JSON]
           API.Types.ProviderPlatform.Management.Driver.ClearDriverFeeReq
      :> Post '[JSON] Kernel.Types.APISuccess.APISuccess
  )
```
how resolve this as this is read only file